geocoder_module/geocoder.py

```diff
@@ -409,6 +413,12 @@ def double_check_countries(
         new_locations = []
 
         for location in locations:
+            if not location:
+                new_locations.append([])
+                continue
+            elif not "name" in location:
+                new_locations.append([])
+                continue
```

Comment:
Make this one line as well?
```suggestion
            if not location or not "name" in location:
                new_locations.append([])
                continue
```